src/app/pages/profile/transaction-information/empty-transaction/empty-transaction.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { EmptyTransactionComponent } from './empty-transaction.component';
```

Comment:
Hey @Jaman-dedy, we should use absolute imports

Reply:
I worked on that @CryceTruly , thanks